### DISCUSSION TITLE
Có dịch Random Forest, Decision tree không?

### DISCUSSION THREAD

**Discussion by lkhphuc** [mlbvn/handson-ml2-vn#8] [Terminology] [answered]
Decision Tree: `Cây quyết định` ?
Random Forest: `Rừng ngẫu nhiên` ?

Mình thấy 2 từ này dịch ra nghe cứ sao sao, tuy nghĩa của mỗi chữ không có gì đặc biệt.

cc: @ngcthuong @rootonchair @nguyenhoa93 @quangnhat185 @minhduc0711 @davidnvq @cuongvng @duythanhvn

**Comment by minhduc0711** [comment]
mình thấy ok

**Comment by rootonchair** [comment]
Cây quyết định em thấy ok. Nhưng Random Forest em sẽ theo hướng để nguyên

**Comment by quangnhat185** [comment]
Mình nghĩ anh em nên thống nhất các thuật toán ML nếu đã dịch thì dịch hết, còn không thì giữ nguyên luôn.

**Comment by minhduc0711** [comment, marked as answer]
mình tạm chốt là dịch nhé mọi người
`Decision Tree`: `Cây quyết định`
`Random Forest`: `Rừng ngẫu nhiên`
